hack/pull_kfserving_upstream.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Usage:
+# This script should be run from kubeflow/manifest repo's root:
+# ./hack/pull_kfserving_upstream.sh
+
+set -ex
+
+# Please edit the following version before running the script to pull new
+# kfserving version.
+export KFSERVING_VERSION=v0.4.0
+export KFSERVING_SRC_REPO=https://github.com/kubeflow/kfserving.git
+
+if [ -d kfserving/upstream ]; then
+    rm -rf kfserving/upstream
+fi
+kpt pkg get $KFSERVING_SRC_REPO/install/$KFSERVING_VERSION kfserving/upstream
+
+# Replace 'kfserving-system' namespace references with 'kubeflow'.
+find kfserving/upstream -name 'kfserving.yaml' -exec \
+    sed -i.bak 's/kfserving-system/kubeflow/' {} +
```

Comment:
@pvaneck @jazzsir recently added a kubeflow overlay in this [PR](https://github.com/kubeflow/kfserving/pull/1103) to directly generate the KFServing release yaml for kubeflow installation.

 There are a few other places you need to change to configure the kubeflow gateway.

Reply:
Oh nice. I see that you will be potentially generating a `kfserving_kubeflow.yaml` in the next release? If so, then this can be removed.

Reply:
we would still need to test and integrate kubeflow for each release,  if user can not wait for kubeflow release they can use the yaml KFServing generates but they are responsible for testing out in their setup